index.html

```diff
@@ -0,0 +1,203 @@
+    <link rel="stylesheet"  href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <a class="logo" href="#">Tiago L. Marin</a>
+            <div class="menu">
+                <img src="/assets/icon-menu.png" alt="Menu Button">
+            </div>
+        </header>
+            <section class="intro">
+                <h1>Hey There.<br>I'm Tiago</h1>
+                <h2>I'm a Software Developer</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+                <ul class="social-icons">
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-gitHub.png" alt="Github Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-linkedin.png" alt="Linked-in  Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-angelist.png" alt="Github Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-twitter.png" alt="Twitter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-medium.png" alt="Github Page">
+                        </a>
+                    </li>
+                </ul>
+            </section>
+            <section class="projects-grid">
+                <div class="grid-title">
+                    <h3>My Recent Works</h3>   
+                </div>
+                <article class="featured">
+                    <img src="/assets/project-featured.jpg" alt="Menu Button">                        
+                    <div class="wrap-card">
+                        <h3 id="article-h">Multi-Post Stories</h3>
+                        <p id="featured-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        <div class="lang">
+                            <button type="button">css</button>
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
```

Comment:
- [ ] [REQUIRED] The list of project technologies should(e.g., "JavasScript", "HTML" and "CSS") a list `ul` and each of them a `li`. Please make changes!